README.md

```diff
@@ -19,7 +19,7 @@ Each package has its own **node_modules** directory that contains symbolic links
 ## Prerequisites
 
 - [Git](https://git-scm.com/)
-- [Node](https://nodejs.org/en/): an installation of the latest security patch of Node 16. The Node installation also includes the **npm** package manager.
+- [Node](https://nodejs.org/en/): an installation of the latest security patch of Node 18. The Node installation also includes the **npm** package manager.
```

Comment:
These prereqs have always seemed confusing to me. Shouldn't they state the range of node versions supported rather than appear to require a specific one?

Reply:
Makes sense to me.